src/bin/commands/images.rs

```diff
@@ -185,7 +195,9 @@ fn remove_images(
         command.arg("--force");
     }
     command.args(images);
-    if execute {
+    if images.is_empty() {
+        Ok(())
+    } else if execute {
         command.run(verbose, false).map_err(Into::into)
     } else {
         println!("{:?}", command);
```

Comment:
this can use the new `command_pretty()` fn

Reply:
i don't see this being done, but it's fine

Reply:
`Command::run` invokes `Command::run_and_get_status` which invokes `Command::print_verbose` which then called `Command::command_pretty`.

Reply:
Oh.... Oh.... Nevermind. I'll cancel this.